zetta_utils/constants.py

```diff
@@ -1,3 +1,4 @@
 """Common project-wide constants."""
 
 DEFAULT_PROJECT = "zetta-research"
+RUN_DATABASE: str | None = "run-db"
```

Comment:
Why is it allowed to be `None`?

Reply:
The emulator image used for firestore doesn't support custom database name, so we have to set it to `None` to indicate default database in tests.